src/functions.js

```diff
@@ -0,0 +1,52 @@
+const listContainer = document.querySelector('.list-container');
+const itemsContainer = document.querySelector('.list-container');
+
+export default class Collection {
+  static tasks = [];
+
+  static createTask = (task) => {
+    const objTasks = {
+      description: task,
+      completed: false,
+      index: Collection.tasks.length + 1,
+    };
+    Collection.tasks.push(objTasks);
+    return objTasks;
+  };
+
+  static showTasks = () => {
+    listContainer.innerHTML = '';
+    Collection.tasks = JSON.parse(localStorage.getItem('tasks'));
+    if (!Collection.tasks) {
+      Collection.tasks = [];
+    } else {
+      Collection.tasks.forEach((element) => {
+        const items = document.createElement('li');
+        items.classList.add('item-list');
+        items.id = element.index - 1;
+        items.innerHTML = `
+          <input type="checkbox" id="id-${element.index}">        
+          <p class="label" contenteditable="true">${element.description}</p>
+          <box-icon class="icon" name='dots-vertical-rounded'></box-icon>
+          <box-icon class="trash-icon d-none" name='trash'></box-icon>`;
+        itemsContainer.appendChild(items);
+      });
+    }
+  };
+
+  static deleteTask = (del) => {
+    let indexArray;
+    Collection.tasks.forEach((element, index) => {
+      if (element.description === del) {
+        indexArray = index;
+      }
+    });
+    Collection.tasks.splice(indexArray, 1);
+    Collection.setLocalStorage();
+    window.location.reload();
+  };
```

Comment:
- [ ] [OPTIONAL] As part of the requirement we need the indexes to be updated on localStorage when you delete an item. if you don't do this you'll end up in a situation where you have multiple tasks with the same index. 

![image](https://user-images.githubusercontent.com/18730106/149599110-c89e6413-2c4d-4045-b442-24c5b3b6364b.png)